brewcask.sh

```diff
@@ -11,7 +11,6 @@ casks+=(
 iterm2
 google-drive
 vmware-horizon-client
-docker
```

Comment:
I understood that we needed to install 'Docker for Mac' rather than the docker brew cask. Have you found otherwise?

https://docs.docker.com/docker-for-mac/docker-toolbox/#setting-up-to-run-docker-for-mac